### DISCUSSION TITLE
ESP32-CAM Water meter - add dot pointer

### DISCUSSION THREAD

**Discussion by petrklic16** [jomjol/AI-on-the-edge-device#2153] [General] [answered]
Hi, 

 i would like to ask you. 
I configured ESP32-CAM for water meter, but i dont know how i can add dot poinet in value - 

![image](https://user-images.githubusercontent.com/38321400/224424662-25d61c30-8cd3-434b-b703-19b1574ce023.png)

Is it possible to do it or i need to do it in Home Assistant?

**Comment by friedpa** [comment, marked as answer]
Set Decimal Shift in the config.ini is the right parameter, you have to set it to -3
Example of my configuration with two digits:

![grafik](https://user-images.githubusercontent.com/42890695/224426745-601b9c66-27a5-42a4-8b2b-94c97936d3d9.png)
